rules.adoc

```diff
@@ -146,40 +143,48 @@ game will be restarted with a kick-off from the team who was scored against.
 [[inside-penalty-area]]
 === Inside the Penalty Area
 
-{++No robots are supposed to be inside any penalty area for more than 15 seconds.
+No robots are supposed to be inside any penalty area for more than 15 seconds.
 After this time, they will be re-spawned on the furthest unoccupied neutral spot
 facing sideways. For this rule to apply, a robot must be inside the penalty area with its center
-of mass.++}
+of mass.
 
-{++For the timer to be reset, the robots must be outside the penalty area for
-more than 2 seconds.++}
+For the timer to be reset, the robots must be outside the penalty area for
+more than 2 seconds.
 
 [[lack-of-progress]]
-=== Lack of progress
+=== Lack of Progress
 
 Lack of progress occurs if there is no progress in the gameplay for a
 reasonable period of time.  Typical
 lack of progress situations are when the ball is stuck between robots, when
 there is no change in ball and robot’s positions, or when the ball is beyond
 detection or reach capability of all robots on the field.
 
-{++If no significant ball movement occurs for 10 seconds++}, the artificial referee will call "lack of progress" and
+If no significant ball movement occurs for 10 seconds, the artificial referee will call "lack of progress" and
 will move the ball to the nearest unoccupied neutral spot. If this does not
 solve the lack of progress, the referee can move the ball to a different
 neutral spot.
 
+{++When placing the ball on a neutral spot, the exact position may vary a little.
+Furthermore, when placing the ball in the center of the field, the ball may be placed
+anywhere within the center circle instead. Consequently, the exact ball position after a
+lack-of-progress situation cannot be predicted.++}
+
 
 [[out-of-bounds]]
-=== Out of bounds
+=== Out of Bounds
+
+There is no "Out of Bounds" rule.
 
-{++There is no "Out of Bounds" rule.++}
+{++If the ball accidentally leaves the field of play, it will be returned into play immediately,
```

Comment:
This might happen to robots as well (we never know!). So, I recommend adding "If the ball or a robot accidentally leaves the field of play..."

Reply:
fixed :-)